### ISSUE TITLE
libvte doesn't display fonts properly on plasma (T9594)

### ISSUE THREAD

**Issue by celticmagic** [getsolus/packages#236]
<div dir="rtl"><b>Joey&nbsp;Riches</b>&nbsp;(<em>&#35;joebonrichie</em>),&nbsp;2021-02-24&nbsp;15:23:22&nbsp;UTC</div><br/>

Affects any terminal using libvte e.g. tilix, gnome-terminal

{F7094722}

**Comment by TraceyC77** [comment]
@joebonrichie , is this still an issue? Forgive me if this is a duplicate notification, I don't know if you got the last one during systems migrations

**Comment by joebonrichie** [comment]
Yes, this is still a problem with an OOTB install of any libvte based terminal on plasma. The solution is to use a custom font.

**Comment by EbonJaeger** [comment]
Here are some observations from further investigation:

1. The issue does not occur on Fedora KDE. That said, they also don't use Hack (or even have it packaged, as far as I can tell). Installing Hack manually and setting it as the font in GNOME Terminal renders as expected.
2. On Solus Plasma, checking the box to use a custom font, but not actually setting one, also fixes the rendering. It appears to be still the same font, but displayed correctly.

This is what it looks like with the incorrect rendering:
![image](https://github.com/getsolus/packages/assets/5157277/449a0a7e-cd73-46bb-9bac-73ae0d4cec27)

And after checking the Custom Font option, without setting a new font:
![image](https://github.com/getsolus/packages/assets/5157277/16b0f0b7-7fc4-4d00-9b00-f0e675fb8875)

This all makes me much less inclined to say this is an upstream issue. I have a feeling that we're missing something, but I don't know enough about font handling to guess where. The VTE package is basically the same between Solus and Fedora. Does anyone else have any ideas on where to start looking?

**Comment by ReillyBrogan** [comment]
For the life of me I cannot seem to figure this one out. @joebonrichie / @ZachBacon ?
